EpikCord/client/client_user.py

```diff
@@ -37,8 +35,8 @@ async def edit(
         if avatar:
             payload["avatar"] = self.client.utils.bytes_to_base64_data(avatar)
         response = await self.client.http.patch("users/@me", json=payload)
```

Comment:
Also, although it is not part of PR, there is inconsistency between here and in HTTPClient, there it does `/...` but here it does `...`, so maybe fix that aswell ?

Reply:
What do you mean? I don't understand

Reply:
In HTTPClient, requests start with `/`, for example `/gateway/bot`,
but here it is `users/@me`

base_uri's default is `https://discord.com/api/v10` which does not end with /, so fix that `users/@me` to `/users/@me` ?

Reply:
I'm lazy, and so naturally HTTPClient will understand where to put `/`, and it will understand this context, so we don't need to have any consistency when making requests

Reply:
I see then.